.eslintrc

```diff
@@ -22,7 +22,6 @@
     "no-unused-vars": ["error", { "argsIgnorePattern": "state|dispatch"}],
     "object-curly-spacing": ["warn", "always"],
     "padded-blocks": ["warn", "never"],
-    "semi": ["warn", "never"],

```

Comment:
Don't we want to force us to use either use the semicolons or remove all of them? If I understand eslint, by removing the rule we're making the semicolon option (aka eslint will not give a shit if it's present or not).

Preferable I would want to have prettier (or a similar tool) to enforce the formatting rules.